composer.json

```diff
@@ -41,7 +42,7 @@
     },
     "flex-require-dev": {
         "symfony/debug-pack": "*",
-        "symfony/maker-bundle": "^1.26.1",
+        "symfony/maker-bundle": "^1.0",
```

Comment:
Why is this version (and `symfony/flex`) lowered?

Reply:
Consistency maybe? @webhdx

Reply:
Yes, I aligned it with `symfony/website-skeleton`

Reply:
It will pick up latest version anyway while we are tagging.

Reply:
I'm ok with it, just curious.